src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -285,7 +285,6 @@
     <Content Include="Views\Partials\Grid\Bootstrap3-Fluid.cshtml" />
     <Content Include="Umbraco\Views\Preview\Index.cshtml" />
     <Content Include="Umbraco\Views\web.config" />
-    <Content Include="Views\Partials\BlockList\Default.cshtml" />
```

Comment:
Although this Razor view is removed, it's still left untouched within existing projects (that installed a previous Umbraco 8.7+ version) and because the `BlockListTemplateExtensions` methods are only obsoleted, rendering of the Block List items will continue to work.